### PR TITLE
Update postop.tsx

### PR DESCRIPTION
**Pull request by drzek**
Updated post op copy

### CODE REVIEW

File: pages/postop.tsx [hunk 1]
```diff
@@ -33,48 +33,78 @@ function PostOp() {
...
+              Check BGL and ketone every 2 hours until eating and drinking normally. Resume SGLT2i when patient returns to full oral intake. Advise patients
+              to check blood glucose and ketones at home if feeling unwell in the week following surgery.
+            </p>
+            
+            <p>
+              Consider overnight admission if vomiting / poor oral intake and persistent ketosis. 
+            </p>
+            
+            <p>
+              (If ketones > 1.0, consider 50 ml 50% dextrose and 2-4 units insulin bolus to facilitate ketone clearance.
```
Comment (drzek) [inline pages/postop.tsx]:
```suggestion
              (If ketones {'>'} 1.0, consider 50 ml 50% dextrose and 2-4 units insulin bolus to facilitate ketone clearance. 
```

File: pages/postop.tsx [hunk 2]
```diff
@@ -33,48 +33,78 @@ function PostOp() {
       case 'repeatKetones':
         return (
           <QuestionCard
-            question="Repeat ketones and BGL in PACU"
-            primaryAction={{ copy: 'Ketones > 1.0 mmol/L', action: sendYes }}
-            secondaryAction={{ copy: 'Ketones <= 1.0 mmol/L', action: sendNo }}
-          />
+            question="Repeat Ketones and Glucose in PACU"
+            primaryAction={{ copy: 'Ketones > 1.0', action: sendYes }}  //I think this might be round the wrong way
```
Comment (miamollie) [inline pages/postop.tsx]: Thanks, let's fix this in a separate PR

```suggestion
            primaryAction={{ copy: 'Ketones > 1.0', action: sendYes }}
```

File: pages/postop.tsx [hunk 3]
```diff
@@ -33,48 +33,78 @@ function PostOp() {
...
       case 'daySurgery':
         return (
           <QuestionCard
             question="Day surgery?"
             primaryAction={{ copy: 'Yes', action: sendYes }}
             secondaryAction={{ copy: 'No', action: sendNo }}
-          />
+          >
+            <p>
+              Is that patient planned to be discharged the day of the procedure?
```
Comment (miamollie) [inline pages/postop.tsx]:
```suggestion
              Is the patient expected to be discharged the day of the procedure?
```

File: pages/postop.tsx [hunk 4]
```diff
@@ -33,48 +33,78 @@ function PostOp() {
...
+            </p>            
           </ResultCard>
         );
       case 'DKA':
         return (
-          <ResultCard result="DKA">
-            Suspect DKA. Contact endocrinology, start DKA insulin/dextrose infusion, Consider HDU Bed
+          <ResultCard result="Metabolic Acidosis">
+            <p>
+              Suspect DKA. Contact endocrinology, start DKA insulin/dextrose infusion. Consider HDU Bed for close monitoring.
```
Comment (miamollie) [inline pages/postop.tsx]:
```suggestion
              Suspect DKA. Contact endocrinology, start DKA insulin/dextrose infusion. Consider HDU bed for close monitoring.
```

**Comment by miamollie** [comment]
LGTM :shipit:

**Review by miamollie — APPROVED** [review]

**Pull request merged**